auction-client/src/components/Lot/Lot.tsx

```diff
@@ -0,0 +1,26 @@
+import './Lot.scss';
+
+import lotImg from '../../images/img/lot-img.png'
+import favIcon from '../../images/icons/fav-icon.svg';
+
+interface Props {
+    title: string; 
+  }
+  
+  export const Lot: React.FC<Props> = ({ title }) => {
+    return (
+      <div className="lot">
+        <div className="lot__galery">
+          <img className="lot__galery-img" src={lotImg} alt="Photo of lot" />
```

Comment:
```suggestion
          <img className="lot__galery-img" src={lotImg} alt="lot" />
```
не треба писати фото чи зображення - скрінрідери вже знають, що це фото